0001_initial_schema.up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE Users (
+  id INT NOT NULL UNIQUE,
+  username CHAR(100),
+  password_hash BINARY(64) NOT NULL,
+  first_name CHAR(100),
+  location_city CHAR(100),
+  location_state CHAR(100),
+  is_curator BOOLEAN,
+  email CHAR(100),
+  instagram CHAR(100),
+  facebook CHAR(100),
+  tiktok CHAR(100),
+  whatsapp CHAR(100),
+  twitter CHAR(100),
+  pinterest CHAR(100),
+  last_login TIMESTAMP,
```

Comment:
should any of these fields be not nullable?

Reply:
username and email stand out to me as eligible for not nullable.

Reply:
also will add a last_name field

Reply:
lets go with that!